blazor/diagram/scroll-settings.md

```diff
@@ -101,7 +101,92 @@ The Diagram control provides the following event for the scroll settings.
 
 }
 ```
+## AutoScroll 
+The auto-scroll feature automatically scrolls the diagram whenever the node is moved beyond its boundaries. Auto-scroll is enabled during node dragging and  node resizing, and multiple-selection operations.
 
+The diagram event [OnAutoScrollChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_OnAutoScrollChange) gets triggered when the autoscroll (scrollbars) is changed, and you can do your own customization in this event. 
```

Comment:
The diagram event [OnAutoScrollChange](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_OnAutoScrollChange) gets triggered when the auto-scroll (scrollbars) is changed, and you can do your customization in this event.